parquet-common/src/main/java/org/apache/parquet/hadoop/metadata/ColumnPath.java

```diff
@@ -48,6 +59,10 @@ public static ColumnPath get(String... path){
     return paths.canonicalize(new ColumnPath(path));
   }
 
+  public static ColumnPath getLowerCase(String... path){
+    return pathsLowerCase.canonicalize(new ColumnPath(path));
+  }
```

Comment:
It seems that the toLowerCase can be applied to path directly and then call the existing get(). There's no need for a separate Canonicalizer